modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * osd_quickstart/osd-quickstart.adoc
+// * osd_cluster_create/creating-an-aws-cluster.adoc
+// * osd_cluster_create/creating-a-gcp-cluster.adoc
+
+ifeval::["{context}" == "osd-getting-started"]
+:osd-quickstart:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-aws"]
+:osd-on-aws:
+endif::[]
+ifeval::["{context}" == "osd-creating-a-cluster-on-gcp"]
+:osd-on-gcp:
+endif::[]
+
+:_content-type: PROCEDURE
+ifdef::osd-quickstart[]
+[id="create-cluster-red-hat-account_{context}"]
+= Creating a cluster with a Red Hat cloud account
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+[id="osd-create-aws-cluster-red-hat-account_{context}"]
+= Creating a cluster on AWS with a Red Hat cloud account
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+[id="osd-create-aws-cluster-ccs_{context}"]
+= Creating a cluster on GCP with a Red Hat cloud account
+endif::osd-on-gcp[]
+
+Through {OCM}, you can create an {product-title} cluster 
+ifdef::osd-on-aws[]
+on {AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+on {GCP} 
+endif::osd-on-gcp[]
+using a standard cloud provider account owned by Red Hat.
+
+.Procedure
+
+. Log in to {console-redhat-com} and click *Create Cluster*.
+
+. In the *Cloud* tab, click *Create Cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. In the *Billing model* dialog, configure the subscription type and infrastructure type:
+.. Select a subscription type.
+.. Select the *Red Hat cloud account* infrastructure type to deploy {product-title} in a cloud provider account that is owned by Red Hat.
+.. Click *Next*.
+
+ifdef::osd-quickstart[]
+. Select *Run on Amazon Web Services* or *Run on Google Cloud Platform* to choose a cloud infrastructure provider 
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+. Select *Run on Amazon Web Services* 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+. Select *Run on Google Cloud Platform* 
+endif::osd-on-gcp[]
+and click *Next*.
+
+. On the *Cluster details* page, provide a name for your cluster and specify the cluster details:
+.. Add a *Cluster name*.
+.. Select a cluster version from the *Version* drop-down menu.
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Select a *Persistent storage* capacity and state the number of *Load balancers* that you require for your cluster.
```

Comment:
I think we need to clarify the Load Balancers option here.  It is not clear to a user what they would put here.  They WILL be guessing.  We need to either link to docs that talk about what these load balancers are for, or tell them in the tool tip popup.  They will almost definitely be thinking "How many LB does this service use, where is the arch diagram?".  

These LBs are really for accessing the cluster over non-http/tcp ports.  For example, if they have a service running in the cluster on port 12345, then they would need at least one.

Reply:
Thank you for highlighting this. I have subsequently split the persistent storage and load balancer items into separate steps. I have also added references to the [Storage](https://docs.openshift.com/dedicated/osd_policy/osd-service-definition.html#sdpolicy-storage_osd-service-definition) and [Load balancers](https://docs.openshift.com/dedicated/osd_policy/osd-service-definition.html#load-balancers_osd-service-definition) sections in the OSD service definition from the steps and included links to those sections in the additional resources sections in both of the assemblies that use this module.

If further details are required in the docs in addition to what is in those sections, I recommend that we develop that as a separate work item outside of this PR.